perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ResourcesManagerMethod.java

```diff
@@ -24,7 +24,7 @@
 
 		@Override
 		public Resource call(ApiCaller ac, Deserializer parms) throws PerunException {
-			return ac.getResourceById(parms.readInt("id"));
+			return ac.getResourcesManager().getResourceById(ac.getSession(), parms.readInt("id"));
```

Comment:
A would allow this method for ``FACILITYADMIN`` too. They probably use version with RichResource from GUI, but its not consistent with expected logic that facility admin can retrieve resources of his facility (by any identifier).